lib/plugins/entities.js

```diff
@@ -338,6 +339,12 @@ function inject (bot) {
     // entity status
     const entity = fetchEntity(packet.entityId)
     const eventName = entityStatusEvents[packet.entityStatus]
+
+    if (eventName === 'entityHandSwap' && entity.equipment) {
+      [entity.equipment[0], entity.equipment[1]] = [entity.equipment[1], entity.equipment[0]]
```

Comment:
is this valid syntax?

Reply:
Just asking, is it possible to check if an item in the offhand like a shield is being used?

Reply:
This isn't really the place to ask for help. I'd recommend you join the Discord: https://discord.com/invite/GsEFRM8
Or if that's not possible for you you can use the discussions: https://github.com/PrismarineJS/mineflayer/discussions

Anyway since you've already asked, you have to use the entity metadata like so:
```js
function getActiveHand (entity) {
    switch (entity.metadata[8]) {
        case undefined: 
        case 0:
        case 2: return 'none'
        case 1: return 'hand'
        case 3: return 'offhand'
    }
}
```